signer/universal/universal_test.go

```diff
@@ -23,6 +23,60 @@ var validLocalConfig = &config.Config{
 	},
 }
 
+var validMinimalUniversalConfig = `
+{
```

Comment:
The case that this covers is when there are two profiles. One with an auth_remote, and one with no auth_remote. Can you add this case?